### PR TITLE
Update CODEOWNERS, add more descriptions

### PR DESCRIPTION
**Pull request by bennyborn**

### CODE REVIEW

**Comment by bennyborn** [comment]
@fritzmg Could you, or anyone else authorized, please merge these changes?

Also: Any idea why the linter forced us to provide the property `homepage` for our package `contao-isotope-amazon-pay` and only on this specific package?

**Comment by fritzmg** [comment]
> Also: Any idea why the linter forced us to provide the property `homepage` for our package `contao-isotope-amazon-pay` and only on this specific package?

There is no publicly available packaged called `numero2/contao-isotope-amazon-pay`. For private packages you need to provide the `homepage` key.

File: meta/numero2/contao-storelocator/en.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+en:
+    title: Contao StoreLocator
+    description: Extension for managing stores (or in common address data) and providing a frontend-search based on geo data.
```
Comment (fritzmg) [inline meta/numero2/contao-storelocator/en.yml:3]:
```suggestion
    description: Extension for managing stores (or address data in general) and providing a frontend-search based on geo data.
```

File: meta/numero2/contao-isotope-simple-erp/en.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+en:
+    title: Simple Isotope stock management
+    description: The simplest ERP system for Isotope eCommerce. Products can be assigned a quota which is decreased with each order. Optionally, products can be automatically switched invisible as soon as no more stock is available.
```
Comment (fritzmg) [inline meta/numero2/contao-isotope-simple-erp/en.yml:3]:
```suggestion
    description: The simplest ERP system for Isotope eCommerce. Products can be assigned a quota which is decreased with each order. Optionally, products can be automatically switched to be invisible as soon as no more stock is available.
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — APPROVED** [review]

**Pull request merged**